packages/components/src/templates/next/components/internal/Tag/Tag.tsx

```diff
@@ -0,0 +1,16 @@
+import { PropsWithChildren } from "react"
+
+export const Tag = (
+  props: PropsWithChildren<
+    React.DetailedHTMLProps<
+      React.HTMLAttributes<HTMLParagraphElement>,
+      HTMLParagraphElement
+    >
+  >,
+) => {
+  return (
+    <div className="flex w-fit items-center justify-center gap-2 rounded-full bg-base-canvas-backdrop px-1.5 py-0.5 text-base-content-subtle">
```

Comment:
question - why do we need to `flex` and `gap-2` these?

Reply:
removed - double checked storybook to ensure no changes